src/bin/vip-import-media.js

```diff
@@ -66,15 +92,68 @@ command( {
 	appContext: true,
 	appQuery,
 	envContext: true,
-	requiredArgs: 1,
 	module: 'import-media',
-	requireConfirm: 'Are you sure you want to import the contents of the provided file?',
-	skipConfirmPrompt: true,
+	requireConfirm: `
+${ chalk.bold( 'NOTE: If the provided archive\'s directory structure begins with `/wp-content/uploads`,' ) }
+${ chalk.bold( 'then we will extract only the files after that path and import it. Otherwise, we will' ) }
```

Comment:
```suggestion
${ chalk.red( chalk.bold( 'we will extract only the files after that path and import it. Otherwise, we will' ) ) }
```

Reply:
@hanifn , looks good, left a suggestion

Reply:
>  we can wrap all three lines in red chalk

Done in 09c0b37659ba06e90a9b2209e26cb6c1baf5edf9